plugin/evm/upgrade/acp176/acp176.go

```diff
@@ -18,24 +18,24 @@ import (
 )
 
 type Config struct {
-	MinGasPrice gas.Price // M
+	MinTargetPerSecond            uint64    // P
+	MinGasPrice                   gas.Price // M
+	TargetToMax                   uint64    // multiplier to convert from target per second to max per second
+	TimeToFillCapacity            uint64    // in seconds
+	TargetToPriceUpdateConversion uint64
 }
 
 const (
-	MinTargetPerSecond  = 1_000_000                                 // P
+	MaxTargetExcessDiff = 1 << 15                                   // Q
 	TargetConversion    = MaxTargetChangeRate * MaxTargetExcessDiff // D
-	MaxTargetExcessDiff = 2 ^ 15                                    // Q
 
-	TimeToFillCapacity            = 10   // in seconds
-	TargetToMax                   = 2    // multiplier to convert from target per second to max per second
-	TargetToPriceUpdateConversion = 87   // 87s ~= 60s * ln(2) which makes the price double at most every ~60 seconds
-	MaxTargetChangeRate           = 1024 // Controls the rate that the target can change per block.
```

Comment:
this is something we cannot parameterize because EVM also imposes the same limit.

Reply:
This is something we can parameterize

Reply:
isn't this the same value for https://github.com/ava-labs/subnet-evm/blob/master/params/protocol_params.go#L36?